### DISCUSSION TITLE
What is the difference between "Impact" and "Output"?

### DISCUSSION THREAD

**Discussion by cmeessen** [research-software-directory/RSD-as-a-service#454] [Q&A] [answered]
This was the question we received from in a [feedback](https://github.com/hifis-net/RSD-as-a-service/issues/24), and I think this is a very good question that could be answered here :)

**Comment by jmaassen** [comment, marked as answer]
On the project pages we see 'Output' as things which are produced by the project itself. These can be papers, software, datasets, blogs, etc.  
As these are produced by the project itself, they don't directly show the impact of the project, only how productive it has been. 

Once others outside of the project start reusing (or mentioning) this output, it becomes impact. This could be papers by others re-using your software or data, articles or videos in the press describing the results, policy documents based on these results, etc. 

We typically only mention things as impact which are a bit broader than just a straightforward citation in a related work section of a paper. For example in the "googling the cancer genome" project (https://research-software-directory.org/projects/gtcg) we created a workflow for detecting structural variants in DNA which may be a source of cancer. To our surprise, this tool was also used in a science paper to detect DNA changes in elephant populations which lose their tusks in a reaction to poaching. We have other examples of projects mentioned in the television news and newspaper articles, getting awards, etc.
